server/server.go

```diff
@@ -31,6 +31,15 @@ import (
 	"github.com/weaveworks/common/signals"
 )
 
+// Dial connects to the address on the named network
+const (
+	// DefaultNetwork  the host resolves to multiple IP addresses,
+	// Dial will try each IP address in order until one succeeds
```

Comment:
Nit-picking: I would expect Dial to try ipv4 and ipv6 in parallel, per RFC 8305.
But it's irrelevant since we don't call Dial